packages/procdot.vm/tools/chocolateyinstall.ps1

```diff
@@ -0,0 +1,43 @@
+$ErrorActionPreference = 'Stop'
+Import-Module vm.common -Force -DisableNameChecking
+
+$toolName = 'procdot'
+$category = 'Utilities'
+$toolDir = Join-Path ${Env:RAW_TOOLS_DIR} $toolName
+
+$zipUrl = 'https://procdot.com/download/procdot/binaries/procdot_1_22_57_windows.zip'
+$zipSha256 = '927cd36dbb4dc0be94afb6021ca7f747dd3f17aad383583bc71aa6e36a762849'
+$zipPassword = "procdot"
+
+try {
+    # Remove files from previous zips for upgrade
+    VM-Remove-PreviousZipPackage ${Env:chocolateyPackageFolder}
+
+    # Download and unzip
+    $packageArgs = @{
+        packageName    = ${Env:ChocolateyPackageName}
+        unzipLocation  = $toolDir
+        url            = $zipUrl
+        checksum       = $zipSha256
+        checksumType   = 'sha256'
+        url64bit       = $zipUrl
+        checksum64     = $zipSha256
```

Comment:
this is only needed if there is 64 version
```suggestion
```

Reply:
There is both 64 bit and 32 bit versions in the same download. 🙂